docker/smartnet-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       - ALLOW_PRIVATE_IPV4=true
       - BASE_PATH=/data/$NODE_ID
       - CHAIN=/data/chainspec.smartnet.json
-      - CUSTOM_ARGS=-lruntime::contracts=debug
+      - CUSTOM_ARGS=-lerror,runtime::contracts=debug
```

Comment:
When running smart node, only those logs show, just double checking if it expected:

```
marcin@marcin-Latitude-3520 ~/g/aleph-node (contracts/e2e-tests)> ./.github/scripts/run_smartnet.sh                                                                                                                                                                        (base) 
Starting smartnode ... done
Attaching to smartnode
smartnode    | Purging chain (/data/chainspec.smartnet.json) at path /data/5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH
smartnode    | "/data/5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH/chains/a0smnet/db/full" removed.
smartnode    | "/data/5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH/backup-stash/0" removed.
smartnode    | 2022-07-25 06:43:56.382 ERROR tokio-runtime-worker sc_network::service: 💔 The bootnode you want to connect provided a different peer ID than the one you expect: `12D3KooWMRFzkiBdNJyggkBaQL92aVb2sDcfVTu9F8eUT31uukom` with `12D3KooWMRFzkiBdNJyggkBaQL92aVb2sDcfVTu9F8eUT31uukom`:`Dialer { address: "/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWMRFzkiBdNJyggkBaQL92aVb2sDcfVTu9F8eUT31uukom", role_override: Dialer }`. 
```

Reply:
I noticed, it's weird because the logged addresses actually are the same - but it's harmless because there's one node anyway